plugins/source/azure/client/services/search.go

```diff
@@ -10,10 +10,10 @@ import (
 )
 
 type SearchClient struct {
-	Service SearchServiceClient
+	Services SearchServicesClient
```

Comment:
As in other places this change is to keep naming conventions consistent between services